htdocs/hrm/position.php

```diff
@@ -160,8 +160,8 @@
 	$backurlforlist = dol_buildpath('/hrm/position_list.php', 1);
 	//$backtopage = dol_buildpath('/hrm/position.php', 1) . '?fk_job=' . ($fk_job > 0 ? $fk_job : '__ID__');
 
-	if (empty($backtopage) || ($cancel && empty($fk_job))) {
-		if (empty($backtopage) || ($cancel && strpos($backtopage, '__ID__'))) {
+	if (!empty($backtopage) || ($cancel && empty($fk_job))) {
```

Comment:
If backtopage is not empty, there is not reason to set it.
Also reference template myobject_card.php use the code   if (empty($backtopage) 
Why changing this ?

Reply:
Nous souhaitions avoir une redirection vers la card après la création et une redirection vers la liste lorsque l'action était annulée comme avec "Produits" ou autres.

Reply:
This is the goal of the backtopage. When this parameter is set and forced, the backtopage must be used in priority.
So everypage can decide where will be the backto when calling the page hrm/position.php

Reply:
delete $backtopage  on button params on list for create and replacing !empty $backtopage to empty $backtopage for card